plugin-tester-scala/src/test/scala/example/myapp/helloworld/GreeterServiceSpec.scala

```diff
@@ -0,0 +1,59 @@
+package example.myapp.helloworld
+
+import scala.concurrent.Await
+
+import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpecLike}
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+import akka.actor.ActorSystem
+import akka.grpc.GrpcClientSettings
+import akka.stream.ActorMaterializer
+import com.typesafe.config.ConfigFactory
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.time.Span
+
+import example.myapp.helloworld.grpc._
+
+class GreeterSpec
+  extends Matchers
+    with WordSpecLike
+    with BeforeAndAfterAll
+    with ScalaFutures {
+
+  implicit val patience = PatienceConfig(5.seconds, Span(100, org.scalatest.time.Millis))
+
+  val serverSystem: ActorSystem = {
+    // important to enable HTTP/2 in server ActorSystem's config
+    val conf = ConfigFactory.parseString("akka.http.server.preview.enable-http2 = on")
+      .withFallback(ConfigFactory.defaultApplication())
+    val sys = ActorSystem("GreeterServer", conf)
+    val bound = new GreeterServer(sys).run()
+    // make sure server is bound before using client
+    bound.futureValue
+    sys
+  }
```

Comment:
One thought is whether we really need TLS for the tests here, I'm pretty sure it has test coverage elsewhere (interop tests)

Reply:
it was intended as a smoke test (well, copied from Quickstart) to verify that our build files here are correct (and capture the knowledge of how to run tests from sbt/gradle/maven with the ALPN agent)